### PR TITLE
v6/ui: add renames and text changes for body header etc

### PR DESCRIPTION
**Pull request by rsteubs**
### Feature

Updating texts according to V6 specifications:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "build:history": "ROUTER_MODE=history npm run build",
     "test": "npm run update:lists && jest --clearCache && npm run test:unit",
     "codecov": "npm run test && codecov",
-    "dev": "npm run update && npm run update:mainlists && vue-cli-service serve --watch",
+    "juststart": "vue-cli-service serve --watch",
```
Comment (olgakup) [inline package.json:14]: whats this new thing?

File: package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
...
+    "dev": "npm run update && npm run juststart",
```
Comment (olgakup) [inline package.json:15]: and not sure why this was changed?

File: src/core/components/TheWrapperDapp.vue [hunk 3]
```diff
@@ -89,7 +89,7 @@ export default {
   methods: {
     onClose() {
       this.$router.push({
-        name: 'Dapps'
+        name: 'DApps'
```
Comment (olgakup) [inline src/core/components/TheWrapperDapp.vue:92]: can you revert this back please? this is actual route name  and they all should be lower case

File: src/core/router/routes-wallet.js [hunk 4]
```diff
@@ -44,7 +44,7 @@ export default {
       beforeEnter: swapRouterGuard
     },
     {
-      path: 'dapps',
+      path: 'DApps',
```
Comment (olgakup) [inline src/core/router/routes-wallet.js:47]: can you revert this back please? this is actual route name and they all should be lower case

**Review by olgakup — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (rsteubs) [reply package.json:15]: [package.json](https://github.com/MyEtherWallet/MyEtherWallet/blob/v6/develop/package.json) in v6/develop

File: package.json (hunk 1 above)
Comment (rsteubs) [reply package.json:14]: [package.json](https://github.com/MyEtherWallet/MyEtherWallet/blob/v6/develop/package.json) in v6/develop, just keeping them in sync

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**